Bottom: ed21d177d20a25a0cf9aa08cdaf0d886d4c9d64e
Top:    d4b7e6e7b498428c03282976b1acacf7e0431f47
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 00:04:33 +0100

Refresh of autoroute-c-fixup-warnings-due

---

diff --git a/src/autoroute.c b/src/autoroute.c
index c799e93..63ab15c 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -4751,7 +4751,9 @@ RouteAll (routedata_t * rd)
 	      p->flags.is_bad = 0;
 	      if (!p->flags.fixed)
 		{
+#ifndef NDEBUG
 		  bool del;
+#endif
 		  assert (!p->flags.homeless);
 		  if (rip)
 		    {
@@ -4773,10 +4775,14 @@ RouteAll (routedata_t * rd)
 		    {
 		      if (TEST_FLAG (LIVEROUTEFLAG, PCB))
 			ripout_livedraw_obj (p);
+#ifndef NDEBUG
 		      del =
+#endif
 			r_delete_entry (rd->layergrouptree[p->group],
 					&p->box);
+#ifndef NDEBUG
 		      assert (del);
+#endif
 		    }
 		  else
 		    {
